tests/testthat/test_system.R

```diff
@@ -129,25 +129,31 @@ test_that("countycode", {
 })
 
 test_that("js_glue_transformer", {
-  v <- c(T,F,NA)
-  res <- glue::glue("{v}", .transformer=js_glue_transformer)
+  exploratory_env <- new.env()
+  exploratory_env$v <- c(T,F,NA)
+  res <- glue_exploratory("@{v}", .transformer=js_glue_transformer)
   expect_equal(as.character(res), "true, false, null")
+  exploratory_env$v <- 1
+  res <- glue_exploratory("{a: {x: @{v}}}", .transformer=js_glue_transformer)
+  expect_equal(as.character(res), "{a: {x: 1}}")
 })
 
-test_that("odbc_glue_transformer", {
-  v <- c(1,2,3)
-  res <- glue::glue("{v*}", .transformer=odbc_glue_transformer)
+test_that("sql_glue_transformer", {
+  exploratory_env <- new.env()
+  exploratory_env$v <- c(1,2,3)
+  res <- glue_exploratory("@{ v }", .transformer=sql_glue_transformer)
   expect_equal(as.character(res), "1, 2, 3")
-  v <- c("a","b","c")
-  res <- glue::glue("{v*}", .transformer=odbc_glue_transformer)
+  exploratory_env$v <- c("a","b","c")
+  res <- glue_exploratory("@{ `v` }", .transformer=sql_glue_transformer)
   expect_equal(as.character(res), "'a', 'b', 'c'") # Not sure if this behavior works for all types of databases.
 })
 
 test_that("bigquery_glue_transformer", {
-  v <- c(1,2,3)
-  res <- glue::glue("{v*}", .transformer=bigquery_glue_transformer)
+  exploratory_env <- new.env()
+  exploratory_env$v <- c(1,2,3)
+  res <- glue_exploratory("@{ v }", .transformer=bigquery_glue_transformer)
   expect_equal(as.character(res), "1, 2, 3")
-  v <- c("a","b","c")
-  res <- glue::glue("{v*}", .transformer=bigquery_glue_transformer)
-  expect_equal(as.character(res), '"a", "b", "c"')
+  exploratory_env$v <- c("a","b","c")
+  res <- glue_exploratory("@{ `v` }", .transformer=sql_glue_transformer)
```

Comment:
```suggestion
   res <- glue_exploratory("@{ `v` }", .transformer=bigquery_glue_transformer)
```

I think this is testing bigquery_glue_transformer, not sql_glue_transformer.

Reply:
Thank you for catching this. Corrected it.